lib/daysOfTheWeek.js

```diff
@@ -9,3 +9,16 @@ module.exports.getDayName = (date) => {
 
   return ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'][passedDate.getDay()];
 };
+
+module.exports.isWeekend = (date) => {
+  if (Object.prototype.toString.call(date) !== '[object Date]' && Number.isNaN(Number(date))) {
+    throw new Error('Date argument must be a Date object or a Javascript epoch timestamp');
+  } else if (Array.isArray(date) || date === null || date === true || date === false) {
+    throw new Error('Date argument must be a Date object or a Javascript epoch timestamp');
+  }
```

Comment:
These lines are the same as in the `getDayName` function.  This suggest maybe they should be abstracted into their own function, so this code can be isolated in one place instead of being copied and pasted all over.  That way, if we need to change this behavior later, we only have to change it one place.  We can also be confident that the behavior is consistent across our app.